src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -1488,6 +1503,8 @@ internal partial class BoundRangeVariable
     {
         protected override OperationKind ExpressionKind => OperationKind.None;
 
+        protected override ImmutableArray<IOperation> Children => ImmutableArray.Create<IOperation>(this.Value);
```

Comment:
>protected override ImmutableArray<IOperation> Children => ImmutableArray.Create<IOperation>(this.Value); [](start = 8, length = 104)

This doesn't feel right. It looks like the value doesn't correspond to anything explicitly present in source.